packages/terra-overlay/src/Overlay.module.scss

```diff
@@ -125,3 +125,4 @@
     }
   }
 }
+/* stylelint-enable */
```

Comment:
I don't see a configuration for `declaration-property-value-blacklist` inside our eslint config. This is probably unnecessary now